pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -532,6 +549,9 @@ func (r *ReconcilePublishingStrategy) ensurePatchableSpec(reqLogger logr.Logger,
 			} else if field == IngressControllerNodePlacement {
 				// If the NodePlacement doesn't match, replace the existing spec with the desired Spec
 				ingressController.Spec.NodePlacement = desiredIngressController.Spec.NodePlacement
+			} else if field == IngressControllerEndPoint {
+				// If the NodePlacement doesn't match, replace the existing spec with the desired Spec
```

Comment:
We may want to add comments for `EndpointPublishingStrategy` instead  `NodePlacement`